packages/pds/src/sql-blockstore.ts

```diff
@@ -1,17 +1,23 @@
 import { IpldStore } from '@atproto/repo'
 import { CID } from 'multiformats/cid'
 import Database from './db'
+import { IpldBlock } from './db/tables/ipld-block'
+import { IpldBlockCreator } from './db/tables/ipld-block-creator'
 
 export class SqlBlockstore extends IpldStore {
+  staged: Map<string, Uint8Array>
```

Comment:
Would be really neat if multiformats/cid gave `CID`s reference equality, and you could meaningfully write `Map<CID, Uint8Array>`.

Reply:
yeah i've had a rough time with CID equality in the past :/ you can even see with CidSet that I have to convert them to strings to get them deduped.